### PR TITLE
WIP: New app page

### PR DESCRIPTION
**Pull request by andrey-p**
This is part of an internal PR.

### CODE REVIEW

**Comment by moollaza** [comment]
@andrey-p is this still WIP? If so please add the WIP label, or assign to and /cc someone to review

File: share/site/duckduckgo/app.tx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                <img class="app__device__frame js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-device-grey.png"/>
+
+                <noscript>
+                    <img class="app__device__screen js-lazyload no-js__hide" src="/assets/app/screens/android-screenshot-favorites.jpg"/>
+                </noscript>
+                <img class="app__device__screen js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-screenshot-favorites.jpg"/>
+            </div>
+            <div class="app__device app__device--android">
+                <noscript>
+                    <img class="app__device__frame js-lazyload no-js__hide" src="/assets/app/screens/android-device-grey.png"/>
```
Comment (bsstoner) [inline share/site/duckduckgo/app.tx]: Don't need any `js-` classes if it's in a `<noscript>` tag.

File: share/site/duckduckgo/app.tx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    </div>
+</div>
+<div class="app app--features blk blk--alt">
+    <div class="app--features__cw cw--c">
+		<div class="app__row">
+			<div class="app__key third half--screen-m">
+				<noscript>
+					<img class="app__key__icon" src="/assets/about/icons/search-alt.png" />
+				</noscript>
+				<img class="app__key__icon js-lazysvg no-js__hide" src="" data-src="/assets/about/icons/search-alt">
```
Comment (bsstoner) [inline share/site/duckduckgo/app.tx]: I wonder if we can figure out a better way to handle this. It sucks that we have to send the same image markup down twice...

**Comment by bsstoner** [comment]
:+1: Looks good!

**Pull request merged**